devtools-ci-index.yaml

```diff
@@ -3102,6 +3102,30 @@
             ci_project: 'devtools'
             ci_cmd: '/bin/bash cico_run_coverage.sh'
             timeout: '25m'
+        - '{ci_project}-{git_repo}-build-master':
+            git_repo: admin-console
+            ci_project: 'devtools'
+            ci_cmd: '/bin/bash cico_build_deploy.sh'
+            saas_git: saas-openshiftio
+            timeout: '20m'
+            extra_target: rhel
```

Comment:
do we have repository for rhel image on quay already?

Reply:
ah I see, we do https://quay.io/repository/openshiftio/rhel-fabric8-services-admin-console